### DISCUSSION TITLE
"Document update conflicts" on usage quota document when there are repeated data providers on same page

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6777] [Help]
Problem:
I have a home screen that presents a few Report tables: 1 table showing Report in progress, 1 table showing Report completed etc. So there are multiple repeated data providers with slightly different filters. 

This screen used to be working fine in older Budibase version. In 1.1.7, I keep seeing "Document update conflicts" error in both builder design view and live app view, whenever the page is loaded (no user action). 

I can reproduce this problem easily in a blank new app, as long as there are a few data providers, as described below.

Version:
1.1.7 Self Hosted
Also happens sometimes on Budibase Cloud 1.1.13

Similar issue:
I found this https://github.com/Budibase/budibase/issues/5728 but it is closed

Steps to reproduce:
1) create a blank app
2) create a Report Table
3) create a few rows
4) create  a data provider and a Report table
5) duplicate the data provider a few times
6) now reload the builder design view a few times, "Document update conflicts" error will show from time to time. 

Demo:
[test multiple data providers-export-1658138064380.txt](https://github.com/Budibase/budibase/files/9131093/test.multiple.data.providers-export-1658138064380.txt)

Expected: 
no "Document update conflicts" error, even if there are repeated data providers on same page

Actual:
"Document update conflicts" when there are repeated data providers on same page

Error:
```
@budibase/server: ERROR [1658209753271] (99003 on MacBook-Pro.local): Document update conflict.
@budibase/server:     Error
@budibase/server:         at generateErrorFromResponse (/Users/User/Documents/Codes/projects2022/BudibaseFork/packages/backend-core/node_modules/pouchdb/lib/index.js:694:18)
@budibase/server:         at /Users/User/Documents/Codes/projects2022/BudibaseFork/packages/backend-core/node_modules/pouchdb/lib/index.js:6816:19
@budibase/server:         at runMicrotasks (<anonymous>)
@budibase/server:         at processTicksAndRejections (node:internal/process/task_queues:96:5)
@budibase/server: Trace: Error
@budibase/server:     at generateErrorFromResponse (/Users/User/Documents/Codes/projects2022/BudibaseFork/packages/backend-core/node_modules/pouchdb/lib/index.js:694:18)
@budibase/server:     at /Users/User/Documents/Codes/projects2022/BudibaseFork/packages/backend-core/node_modules/pouchdb/lib/index.js:6816:19
@budibase/server:     at runMicrotasks (<anonymous>)
@budibase/server:     at processTicksAndRejections (node:internal/process/task_queues:96:5) {
@budibase/server:   message: 'Document update conflict.',
@budibase/server:   error: 'conflict',
@budibase/server:   reason: 'Document update conflict.',
@budibase/server:   docId: 'usage_quota'
@budibase/server: }
```

Screenshot:
![Screenshot 2022-07-18 at 5 56 13 PM](https://user-images.githubusercontent.com/1195573/179488429-93f6075c-82d3-4be6-abd1-cc12330808c1.jpg)

![Screenshot 2022-07-18 at 5 56 42 PM](https://user-images.githubusercontent.com/1195573/179489213-eef9b77b-9415-42a1-8f0e-4233419bc18f.jpg)

**Comment by melohagan** [comment]
Hi @magickaito 

Thanks for sharing. I can see a couple of related issues from this week: https://github.com/Budibase/budibase/issues?q=is%3Aissue+is%3Aopen+conflict

**Reply by magickaito** [reply]
thanks @melohagan. Hopefully the team will resolve it soon, in the mean time, I will try to rollback to earlier version.
